test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -276,6 +276,15 @@ TEST_F(AtenXlaTensorTest, TestSubScalarInPlace) {
   });
 }
 
+TEST_F(AtenXlaTensorTest, TestSymSizes) {
+  ForEachDevice([&](const torch::Device& device) {
+    torch::Tensor a = torch::rand({2, 3}, torch::TensorOptions(torch::kFloat));
+    torch::Tensor xla_a = CopyToDevice(a, device);
+    ASSERT_EQ(a.sym_sizes().at(0).expect_int(), 2);
+    ASSERT_EQ(a.sym_sizes().at(0).is_symbolic(), false);
+  });
+}
+
```

Comment:
should we add a dynamic test? You can call `non_zero` but you would need to make sure https://github.com/pytorch/xla/blob/master/test/cpp/run_tests.sh#L11 is set so `non_zero` won't fallback to cpu.